satellite/satellitedb/migrate.go

```diff
@@ -1211,26 +1211,26 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 				Description: "Add Graceful Exit tables and update nodes table",
 				Version:     56,
 				Action: migrate.SQL{
-					`ALTER TABLE nodes ADD COLUMN exit_loop_completed_at timestamp with time zone;`,
-					`ALTER TABLE nodes ADD COLUMN exit_initiated_at timestamp with time zone;`,
-					`ALTER TABLE nodes ADD COLUMN exit_finished_at timestamp with time zone;`,
+					`ALTER TABLE nodes ADD COLUMN exit_loop_completed_at timestamp;`,
```

Comment:
This is changing migrations. Has it been rolled out or not?

Reply:
Great point.  Looks like the original schema commit is in the v0.21.0 branch going out Wednesday.  I'll fix the migrations.